cid-redirects.json

```diff
@@ -2641,8 +2641,83 @@
+  "/docs/dashboards-new/panels/combo-charts": "/docs/dashboards/panels/combo-charts",
+  "/docs/dashboards-new/panels/line-charts": "/docs/dashboards/panels/line-charts",
+  "/docs/dashboards-new/panels/map-charts": "/docs/dashboards/panels/map-charts",
+  "/docs/dashboards-new/panels/markdown-syntax": "/docs/dashboards/panels/markdown-syntax",
+  "/docs/dashboards-new/panels/pie-charts": "/docs/dashboards/panels/pie-charts",
+  "/docs/dashboards-new/panels/sankey-charts": "/docs/dashboards/panels/sankey-charts",
+  "/docs/dashboards-new/panels/scatter-charts": "/docs/dashboards/panels/scatter-charts",
+  "/docs/dashboards-new/panels/single-value-charts": "/docs/dashboards/panels/single-value-charts",
+  "/docs/dashboards-new/panels/table-charts": "/docs/dashboards/panels/table-charts",
+  "/docs/dashboards": "/docs/dashboards-classic",
+  "/docs/dashboards/about": "/docs/dashboards-classic/about",
+  "/docs/dashboards/share-dashboard-inside-org": "/docs/dashboards-classic/share-dashboard-inside-org",
+  "/docs/dashboards/share-dashboard-outside-org": "/docs/dashboards-classic/share-dashboard-outside-org",
+  "/docs/dashboards/restricted-operators-dashboards": "/docs/dashboards-classic/restricted-operators-dashboards",
+  "/docs/dashboards/troubleshoot-dashboards": "/docs/dashboards-classic/troubleshoot-dashboards",
+  "/docs/dashboards/get-started": "/docs/dashboards-classic/get-started",
+  "/docs/dashboards/get-started/create-dashboard": "/docs/dashboards-classic/get-started/create-dashboard",
+  "/docs/dashboards/get-started/add-dashboard-link": "/docs/dashboards-classic/get-started/add-dashboard-link",
+  "/docs/dashboards/get-started/add-links-text-panels": "/docs/dashboards-classic/get-started/add-links-text-panels",
+  "/docs/dashboards/get-started/add-text-panel": "/docs/dashboards-classic/get-started/add-text-panel",
+  "/docs/dashboards/get-started/dashboard-optimization": "/docs/dashboards-classic/get-started/dashboard-optimization",
+  "/docs/dashboards/get-started/delete-dashboard": "/docs/dashboards-classic/get-started/delete-dashboard",
+  "/docs/dashboards/get-started/markdown-syntax": "/docs/dashboards-classic/get-started/markdown-syntax",
+  "/docs/dashboards/get-started/move-panel-dashboard": "/docs/dashboards-classic/get-started/move-panel-dashboard",
+  "/docs/dashboards/get-started/pause-panel": "/docs/dashboards-classic/get-started/pause-panel",
+  "/docs/dashboards/get-started/resize-panel": "/docs/dashboards-classic/get-started/resize-panel",
+  "/docs/dashboards/get-started/set-data-access-level-dashboard": "/docs/dashboards-classic/get-started/set-data-access-level-dashboard",
+  "/docs/dashboards/get-started/title-panels": "/docs/dashboards-classic/get-started/title-panels",
+  "/docs/dashboards/get-started/zoom-panel": "/docs/dashboards-classic/get-started/zoom-panel",
+  "/docs/dashboards-classic/chart-panel-types": "/docs/dashboards-classic/chart-panel-types",
```

Comment:
@JV0812 it looks like the redirects from this line down are all linking to themselves. Once you fix these, it should work.

Reply:
Thanks Kim! I have updated those.

Reply:
thank you!